sources/procfs.go

```diff
@@ -320,6 +325,14 @@ func (s *lustreProcfsSource) generateClientMetricTemplates() error {
 			{"stats", "inode_permission_total", inodePermissionHelp, s.counterMetric},
 			{"xattr_cache", "xattr_cache_enabled", "Returns '1' if extended attribute cache is enabled", s.gaugeMetric},
 		},
+		"mdc/*": {
+			{"rpc_stats", "rpc_in_flight", rpcsInFlightHelp, s.gaugeMetric},
```

Comment:
That was actually just a typo as I meant to put `rpcs_in_flight`, but I can certainly do `osc*` and `mdc*` if you desire!